go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/jaminyah/GoAjaxForm
```

Comment:
go.mod is the dependency management system recommended for new go projects.
As long as you have a version 1.12 or later version of Go, everything should work as usual transparent to you.